app/assets/javascripts/templates/form.html

```diff
@@ -21,7 +21,7 @@
       Instructions
     </label>
     <div class="col-md-5">
-      <textarea name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
+      <textarea id="instructions" name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
```

Comment:
Since element with 'id' is mandatory when using 'label' with 'for', IDEs such as JetBrains products show us warnings - that's the only reason =)